index.d.ts

```diff
@@ -120,11 +120,17 @@ declare module "hyperapp" {
     | Action<S, P>
     | readonly [action: Action<S, P>, payload: P]
 
+  // An Effecter is the function that runs an effect
```

Comment:
Please change this line to: `// An effecter is the function that runs an effect.`

😅